commit 9f671e58159adea641f76c56d1f0bbdcb3c524ff
Author: Kees Cook <kees@kernel.org>
Date:   Wed Apr 10 08:23:44 2019 -0700

    security: Create "kernel hardening" config area
    
    Right now kernel hardening options are scattered around various Kconfig
    files. This can be a central place to collect these kinds of options
    going forward. This is initially populated with the memory initialization
    options from the gcc-plugins.
    
    Signed-off-by: Kees Cook <keescook@chromium.org>
    Acked-by: Masahiro Yamada <yamada.masahiro@socionext.com>

diff --git a/security/Kconfig b/security/Kconfig
index 1d6463fb1450..7aec8d094ce2 100644
--- a/security/Kconfig
+++ b/security/Kconfig
@@ -249,5 +249,7 @@ config LSM
 
 	  If unsure, leave this as the default.
 
+source "security/Kconfig.hardening"
+
 endmenu